src/openrct2-ui/WindowManager.cpp

```diff
@@ -6,6 +6,7 @@
  *
  * OpenRCT2 is licensed under the GNU General Public License version 3.
  *****************************************************************************/
+#pragma optimize("", off)
```

Comment:
Is this just for debugging or does it need to be there? If the latter, I’d like you to add a comment explaining why.

Reply:
Accident, I'll strip it.

Reply:
Should I also test it with that line removed and with an optimised build?

Reply:
It shouldn't make a huge difference but if you want.